### DISCUSSION TITLE
请问如何重置admin的密码，改密码后忘记了

### DISCUSSION THREAD

**Discussion by SoolyChristy** [AlistGo/alist#2885] [Q&A]
如题

**Comment by SoolyChristy** [comment]
我重新部署之后，登录还是不行
./alist admin 这里的密码就是错的

**Reply by NovaShen0724** [reply]
好像这个问题是通病，参考
https://github.com/alist-org/alist/discussions/721
https://github.com/alist-org/alist/discussions/306
https://github.com/alist-org/alist/discussions/220
https://github.com/alist-org/alist/discussions/2325
如仍无法解决，请回复

**Reply by SoolyChristy** [reply]
没办法解决

**Reply by SoolyChristy** [reply]
我是运行在replit上的

**Reply by SoolyChristy** [reply]
我输入子账号是能进去的，但是admin的不行

**Reply by NovaShen0724** [reply]
如果没办法，就重装吧

**Reply by NovaShen0724** [reply]
https://github.com/alist-org/alist/discussions/2923

**Reply by NovaShen0724** [reply]
请确认当前运行的目录的数据是否可以登录

**Reply by SoolyChristy** [reply]
./alist admin这里的密码登录不了

**Reply by dingodingle** [reply]
重装仍然不行，每次出现的随机密码都不行，也改不了，我是openwrt的装3.33.0-4

**Comment by 12189108** [comment]
alist文件夹下有一个data目录，里面有一个文件叫data.db，请用数据库相关软件打开，如Navicat等，有一个表存储了你的用户名和密码

**Comment by dengyakui** [comment]
Works for me on windows: 
```
 alist.exe password
```
![image](https://user-images.githubusercontent.com/3945919/219932224-30afc91c-5f7f-4f2a-b140-62068e39b161.png)

**Reply by Z-Firmiana** [reply]
helpful

**Comment by fengchen-github** [comment]
cd /opt/alist/
sudo ./alist admin
INFO[2023-06-26 07:12:25] reading config file: data/config.json
INFO[2023-06-26 07:12:25] load config from env with prefix: ALIST_
INFO[2023-06-26 07:12:25] init logrus...
INFO[2023-06-26 07:12:25] admin user's info:
username: admin
password: xxxxxxx
